assets/js/modules/analytics/components/common/AnonymizeIPSwitch.js

```diff
@@ -66,14 +65,13 @@ export default function AnonymizeIPSwitch() {
 					href="https://support.google.com/analytics/answer/2763052"
 					external
 					inherit
-					dangerouslySetInnerHTML={ sanitizeHTML(
-						__( 'Learn more<span class="screen-reader-text"> about IP anonymization.</span>', 'google-site-kit' ),
-						{
-							ALLOWED_TAGS: [ 'span' ],
-							ALLOWED_ATTR: [ 'class' ],
-						}
-					) }
-				/>
+				>
+					{ __( 'Learn more', 'google-site-kit' ) }
+					{ ' ' }
+					<span className="screen-reader-text">
+						{ __( 'about IP anonymization.', 'google-site-kit' ) }
+					</span>
```

Comment:
See above.

Reply:
Updated to use `createInterpolateElement`